plugins/commitment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commitment
+spec:
+  version: v0.1.1
+  homepage: https://github.com/wI2L/kubectl-commitment
+  shortDescription: Compare VerticalPodAutoscaler recommendations to actual resources requests
+  description: |
+    This plugin prints a table that show the percentage increase/decrease
+    of the selected VerticalPodAutoscaler recommendations compared to the
+    targeted controller's pod resource requests.
+  caveats: |
+    The plugin recognize only some well-known controllers such as:
```

Comment:
small typo `recognize` -> `recognizes`

also I think maybe its worth mentioning that the `VerticalPodAutoscaler` needs to be installed in the cluster, especially depending on if the name will contain `vpa` or not

Reply:
Yep, make sense. Will push a fix including @ahmetb recommendation to wrap the description to 50 chars.